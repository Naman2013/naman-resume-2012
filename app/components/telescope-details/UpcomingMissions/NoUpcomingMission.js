import React from 'react';
import PropTypes from 'prop-types';
import { gray } from '../../../styles/variables/colors';

const NoUpcomingMissions = ({ message }) => (
  <div>
    <h4 className="title">{ message }</h4>
    <style jsx>{`
      .title {
        margin: 0;
        text-align: center;
        padding: 50px;
        text-transform: none;
        color: ${gray};
      }
    `}</style>
  </div>
);

NoUpcomingMissions.propTypes = {
  message: PropTypes.string.isRequired,
};

export default NoUpcomingMissions;
