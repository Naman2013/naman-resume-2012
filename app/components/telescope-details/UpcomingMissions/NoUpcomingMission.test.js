import React from 'react';
import renderer from 'react-test-renderer';
import NoUpcomingMission from './NoUpcomingMission';

describe('<NoUpcomingMission />', () => {
  it('should render no mission stuff...', () => {
    const tree = renderer.create(<NoUpcomingMission />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
