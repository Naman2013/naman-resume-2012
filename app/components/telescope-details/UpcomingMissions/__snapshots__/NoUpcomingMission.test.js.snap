// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoUpcomingMission /> should render no mission stuff... 1`] = `
<div
  data-jsx={942546412}
>
  <h4
    className="title"
    data-jsx={942546412}
  />
  <style
    jsx={true}
  >
    
      .title {
        margin: 0;
        text-align: center;
        padding: 50px;
        text-transform: none;
        color: #D9D9D9;
      }
    
  </style>
</div>
`;
